docs/standard/threading/managed-threading-best-practices.md

```diff
@@ -53,7 +53,7 @@ if (Monitor.TryEnter(lockObject, 300)) {
         // Place code protected by the Monitor here.  
     }  
     finally {  
-        Monitor.Exit(this);  
+        Monitor.Exit(lockObject);  
```

Comment:
This change should be applied to the VB version of the code too. I.e. change `Monitor.Exit(Me)` to `Monitor.Exit(lockObject)`.

Reply:
Its already been changed in https://github.com/dotnet/docs/pull/3720.